bin/plan_check

```diff
@@ -295,6 +295,7 @@ def should_run_plan_for(directory: str, plan_path: str) -> bool:
         os.path.commonpath([plan_path, directory]) == plan_path
         and os.path.exists(directory)
         and is_plan_check_enabled(directory)
+        and os.path.isdir(directory)
```

Comment:
The error came from `is_plan_check_enabled`, so seems like this should be above that check.

Reply:
Ah right. Rearranged it so that line will get short circuited